MonstieWash/Assets/Scripts/UI/ScoreUIManager.cs

```diff
@@ -0,0 +1,137 @@
+using JetBrains.Annotations;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.SocialPlatforms.Impl;
+using UnityEngine.UI;
```

Comment:
Just make sure you remove unused libraries.
```suggestion
using System;
using System.Collections;
using System.Collections.Generic;
using TMPro;
using UnityEngine;
using UnityEngine.UI;
```